packages/service-library/tests/test_redis.py

```diff
@@ -249,13 +250,15 @@ async def _inc_counter() -> None:
 
 
 async def test_redis_client_sdks_manager(redis_service: RedisSettings):
-    all_redis_databases: set[RedisDatabase] = set(RedisDatabase)
-    manager = RedisClientsManager(databases=all_redis_databases, settings=redis_service)
+    all_redis_configs: set[RedisManagerDBConfig] = {
+        RedisManagerDBConfig(x) for x in RedisDatabase
```

Comment:
TIP: use `_` instead of `x`

Reply:
In python, by convention `_` means I can be ignored.
Instead of `x` I can use `d` or `db`, the later is better.